pkgs/by-name/mi/miru/package.nix

```diff
@@ -0,0 +1,55 @@
+{ lib
+, fetchurl
+, appimageTools
+}:
+
+appimageTools.wrapType2 rec {
+  pname = "miru";
+  version = "4.5.5";
+  name = "${pname}-${version}";
```

Comment:
We no longer need to generate a name variable since all tools accept pname version.

Reply:
I redefined it since a similar pattern appeared inside the url and install commands.  
Just removed it and squashed everything.